tests/test_examples.py

```diff
@@ -563,3 +563,31 @@ async def test_21_run_playbook():
     event = event_log.get_nowait()
     assert event["type"] == "Shutdown", "8"
     assert event_log.empty()
+
+
+@pytest.mark.asyncio
+async def test_22_print_events():
```

Comment:
Please change the name to test_26_print_events

Reply:
Why tests are numbered like this?

Reply:
They match up with examples that are numbered.

Reply:
Ah, alright, this makes sense.